### PR TITLE
Add questions-policy-compliance.yaml

### PR DESCRIPTION
**Pull request by johndileo**
Create initial draft of questions for Policy & Compliance Practice.

### CODE REVIEW

File: Website/hugo/data/questions-policy-compliance.yaml [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        - B:
+            answer: Yes, for some of the applications
+            weight: 0.2
+        - C:
+            answer: Yes, for at least half of the applications
+            weight: 0.5
+        - D:
+            answer: Yes, for most of the applications
+            weight: 1
+   quality_criteria:
```
Comment (itscooper) [inline Website/hugo/data/questions-policy-compliance.yaml]: This line needs an extra space at the beginning to match the indentation used elsewhere. YAML is horribly picky it turns out :/

Comment (itscooper) [reply Website/hugo/data/questions-policy-compliance.yaml]: Line 130 has 3 spaces indentation - it needs 4.

Comment (johndileo) [reply Website/hugo/data/questions-policy-compliance.yaml]: That's really interesting, given the fact that I didn't change the indentation anywhere in the file. I started with a copy of the YAML content in a different question set, and changed content only to the right of colons...

Comment (johndileo) [reply Website/hugo/data/questions-policy-compliance.yaml]: Space added, and updated file committed.

**Pull request closed without merging**